docs/Aurora/Spark job submit tool usage on aurora.md

```diff
@@ -274,3 +278,10 @@ spark.executor.resource.gpu.amount      1        # each executor has 1 GPU
 spark.executor.instances                36       # 36 (12 x 3) executors in total
 spark.task.resource.gpu.amount          0.125    # Each task uses 1/8 of a GPU. This value should be determined based on your **target task concurrency**. For example, if each node has 12 GPUs and you want 96 concurrent GPU tasks per node, then spark.task.resource.gpu.amount = 12 / 96 = 0.125. In this case, 8 tasks share each GPU, allowing better parallelism for lightweight stages like preprocessing.
 ```
+
+By default, DAOS shuffle is enabled in env_local.sh. If you want to disable DAOS shuffle, you can remove the following lines. 
+```shell
+spark.shuffle.manager                   org.apache.spark.shuffle.daos.DaosShuffleManager
+spark.shuffle.daos.pool.uuid            Intel
+spark.shuffle.daos.container.uuid       spark_shuffle
```

Comment:
"daos pool create Intel spark_shuffle " , It's command for creating shuffle container.